transmart-core-db-tests/build.gradle

```diff
@@ -27,4 +27,7 @@ assets {
     packagePlugin = true
 }
 
+// Force selected tests to always run, even if Gradle thinks they are up to date
```

Comment:
Grails tries to only run tests whose code has changed, but with the TM packaging structure grails doesn't see correctly which tests should run again. This change makes grails always run all selected tests.